RELEASE.md

```diff
@@ -0,0 +1,41 @@
+# Release
+Follow the steps outlined in this document to release a new version of `cloudgov-style`.
+
+### Blah blah
+- Releases should follow [semantic versioning](http://semver.org/).
+- The Github tags for the release should just be the number, no `v` before.
+- The project will be pushed to [npm](https://www.npmjs.com/package/cloudgov-style) and [RubyGems](https://rubygems.org/gems/cloudgov-style).
+
+## Steps
+
+### Preparation
+1. Decide on a "main idea" for the release, what the release is being done for. Look at previous releases for examples.
+2. Decide on whether the release should be a patch, minor or major version update, based on semver.
+3. Create a new release on the Github Releases page.
+  1. Title the release with `{version number} - {main idea}`.
+  2. Use the tag version of the release version number.
+  3. Write a list of changes for the release.
+  4. Create a link titled `Changelog` that uses the Github compare url to compare the current release tag with the last, such as `/compare/0.3.4...0.4.0`.
+
+### Npm something
+1. Ensure the `master` branch is up to date with all code to release.
+2. Checkout a new `release-x.x` branch with the major and minor numbers of the new release version number.
+  - This release branch should have all the changes to be put into the release.
+  - This branch should be cut from `master`.
+3. Update the `package.json` version number to the release version number.
```

Comment:
Thats true, I think the workflow is you put up a PR for the release branch which then gets merged into master, so at that point it has the updated version number.